texar/torch/modules/decoders/transformer_decoders.py

```diff
@@ -292,6 +295,7 @@ def default_hparams():
                 'output_dim': 512,
                 'use_bias': False,
             },
+            'eps': 1e-6,
```

Comment:
Shouldn't the default be `1e-12` in this case?

Reply:
Also, if the user overrides this value, should we also use this `eps` value if `use_gpt_config` is `True`? I kinda feel that the better thing to do is to change our `hparams` for `GPT2*` modules.

Reply:
Changed. Removed the hparam `use_gpt_config`. All GPT2Decoder modules take in eps directly from pretrainedMixin now. Ran GPT2 tests. Also, not sure about the default values, because in `transformer_decoder`, there is a comment which says in PyTorch, the default values for eps is 1e-6 which isn't true because it defaults to 1e-5. Does it only apply to transformer_encoders then?

Reply:
I remember the `eps` was set to `1e-12` to be consistent with some old version of TF, and it was required for certain pre-trained models. IMO I think we should use the default value from PyTorch here, and change hparams in corresponding pre-trained mixins. @gpengzhi Could you help in determining which pre-trained models require an `eps` value of `1e-12`?